packages/salesforcedx-vscode-soql/test/ui-test/soql-ui-smoke.test.ts

```diff
@@ -72,7 +72,7 @@ describe('In project folder, SOQL files should', function() {
 
     for (const action of actions) {
       if (
-        (await action.getAttribute('title')) ===
+        (await action.getAttribute('aria-label')) ===
```

Comment:
Sneaky! How did this get changed?

Reply:
There was an update to the vscode toolbar DOM (this is finding the button in the toolbar when looking at a soql file for toggling the file vs solqbuilder view).  This may have been broken for a while as these tests fail all the time and I just bothered to actually follow up this Monday as part of trust and the failed build.